manual/source/Tutorial/Algorithms/Alignment/PairwiseSequenceAlignment.rst

```diff
@@ -448,7 +448,7 @@ Assignment 5
     * A CIGAR string is a different representation of an alignment.
       It consists of a number followed by an operation.
       The number indicates how many consecutive operations of the same type are executed.
-      Operations can be <tt>M</tt> for match, <tt>S</tt> for mismatch, <tt>I</tt> for insertion and <tt>D</tt> for deletion.
+      Operations can be **M** for match, **S** for mismatch, **I** for insertion and **D** for deletion.
```

Comment:
Can you also please fix the falsely used **S** value. In this case **M** in general is a match or mismatch.
**S** is used to indicate soft clipping not mismatches.
Thank you very much.